lib/Doctrine/DBAL/Driver/SQLSrv/SQLSrvConnection.php

```diff
@@ -139,7 +139,10 @@ public function lastInsertId($name = null)
             return $stmt->fetchColumn();
         }
 
-        return $this->lastInsertId->getId();
```

Comment:
I wanted to remove the custom object altogether but unfortunately the property is `protected` on the connection so I kept it here for BC.